examples/wasmtime/Cargo.toml

```diff
@@ -12,9 +12,11 @@ anyhow = "1.0"
 bitflags = "2.5"
 cfg-if = "1"
 log = { version = "0.4" } #, features = ["kv_unstable"]}
-simple_logger = { version = "5.0", default-features = false, features = ["nightly"]}
-wasmtime = { version = "22.0", default-features = false, features = ["std", "runtime", "cranelift", "threads", "gc", "component-model"] }
+simple_logger = { version = "5.0", default-features = false }
+wasmtime = { version = "22.0", default-features = false, features = ["std", "runtime", "cranelift", "threads"] } #, "gc", "component-model"] }
+zerocopy = { version = "0.7" }
 
 [target.'cfg(target_os = "hermit")'.dependencies]
 hermit = { path = "../../hermit", default-features = false, features = ["smp", "acpi", "pci", "fsgsbase", "fs", "tcp", "dhcpv4", "mmap"] }
 hermit-abi = { path = "../../hermit-abi", default-features = false }
+hermit-sync = { version = "0.1" }
```

Comment:
Can't we use [`std::cell::OnceCell`](https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html) instead?